h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
   max-width: 500px;
+  flex-shrink: 0;
   padding-left: 0;
   padding-right: 0;
 }
 
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
   margin-left: 30px;
   word-wrap: break-word;
```

Comment:
I've left this as `word-wrap` because `overflow-wrap` [doesn't have great browser support](http://caniuse.com/#feat=wordwrap) (maybe I should do a separate PR to replace all `overflow-wrap`s with `word-wrap`? We have a few)